Packs/Microsoft365Defender/ReleaseNotes/4_5_39.md

```diff
@@ -0,0 +1,84 @@
+
+#### Incident Fields
+
+##### New: Microsoft 365 Defender Mailboxes
+
+- New: - Added a new incident field to display mailboxes information mirrored from Microsoft 365 Defender incidents.
+
+##### Microsoft 365 Defender Classification
+
+- - Updated the field type to **"single select"**, possible values:  
+  `TruePositive - MultiStagedAttack`, `TruePositive - MaliciousUserActivity`, `TruePositive - Malware`, `TruePositive - Phishing`, `TruePositive - CompromisedAccount`, `TruePositive - UnwantedSoftware`, `TruePositive - Other`,  
+  `InformationalExpectedActivity - SecurityTesting`, `InformationalExpectedActivity - LineOfBusinessApplication`, `InformationalExpectedActivity - ConfirmedActivity`, `InformationalExpectedActivity - Other`,  
+  `FalsePositive - Clean`, `FalsePositive - NoEnoughDataToValidate`, `FalsePositive - Other`, `Unknown`.
+
+
+##### New: Microsoft 365 Defender Comments
+
+- New: Added a new grid-type field to display comments mirrored from Microsoft 365 Defender incidents. Supports adding comments back to Microsoft 365 Defender with a configurable tag.  
+ <~XSIAM> (Available from Cortex XSIAM %%XSIAM_VERSION%%).</~XSIAM>
+<~XSOAR> (Available from Cortex XSOAR 6.10.0).</~XSOAR>
+
+##### Microsoft 365 Defender Categories count
+
+- Updated the field to dynamically display alert categories count from Microsoft 365 Defender incidents rather than a predefined list of categories.
+
+##### Microsoft 365 Defender Status
+
+- Updated the field type to **"single select"**, possible values: `Active`, `In Progress`, `Resolved`, `Redirected`.
```

Comment:
```suggestion
Updated the field to dynamically display alert categories count from Microsoft 365 Defender incidents rather than a predefined list of categories.

##### Microsoft 365 Defender Status

Updated the field type to **single select**. Possible values are: `Active`, `In Progress`, `Resolved`, `Redirected`.
```